server/migrations/2023-12-13-045001_create-stripe-customers-table/up.sql

```diff
@@ -0,0 +1,14 @@
+-- Your SQL goes here
+ALTER TABLE
+    users
+ADD
+    CONSTRAINT users_email_unique UNIQUE (email);
+
+CREATE TABLE stripe_customers (
+    id UUID NOT NULL UNIQUE PRIMARY KEY,
+    stripe_id TEXT NOT NULL UNIQUE,
+    email TEXT NOT NULL UNIQUE,
```

Comment:
Do we need a constraint with the UNIQUE on this col alr

Reply:
The constraint was on the old users table

I think it's there alr, but I didn't want to check and figure it out so I just added one for sanity